specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/customRecommedations.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
Please check, as your PR is labeled with "NewApiVersionRequired", which means that adding new API paths to an existing API Version are breaking changes.

Please add these changes to a new Preview API Version,or get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.